instrumentation/github.com/gin-gonic/gin/otelgin/gintrace_test.go

```diff
@@ -32,8 +37,9 @@ func TestGetSpanNotInstrumented(t *testing.T) {
 		// Assert we don't have a span on the context.
 		span := trace.SpanFromContext(c.Request.Context())
 		ok := !span.SpanContext().IsValid()
-		assert.True(t, ok)
-		_, _ = c.Writer.Write([]byte("ok"))
+		if !ok {
```

Comment:
Can you keep the PR changes limited to the IP change?

Reply:
Same, consistency with the other test and fixing a tiny bug I stumbled upon.